internal/cli/kraft/build/build.go

```diff
@@ -59,6 +59,33 @@ type BuildOptions struct {
 	statistics map[string]string
 }
 
+// supportedPlatforms lists the supported platforms
+var supportedPlatforms = map[string]bool{
+	"qemu": true,
+	"fc":   true,
+}
+
+// supportedArchitectures lists the supported architectures
+var supportedArchitectures = map[string]bool{
+	"x86_64": true,
+	"arm64":  true,
+	"arm":    true,
+}
```

Comment:
Same here, no need to add them, they are already somewhere, I'll link.

Reply:
file is `machine/platform/platform.go` you can use the stuff in there

also for the architectures it is `unikraft/arch/architecture.go`